src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+
+const spiderWidth = spider.offsetWidth;
+const spiderHeight = spider.offsetHeight;
+
+const wallWidth = wall.offsetWidth;
+const wallHeight = wall.offsetHeight;
```

Comment:
If the elements are not fully loaded, `offsetWidth` and `offsetHeight` might return 0, leading to incorrect positioning. Consider wrapping this code in a `window.onload` event listener to ensure the DOM is fully loaded before executing these calculations.